iOS/UZTickets/src/scan/UZTScanViewController.h

```diff
@@ -0,0 +1,33 @@
+//
+//  UZTScanViewController.h
+//  UZTickets
+//
+//  Created by nekoi on 6/1/15.
+//  Copyright (c) 2015 DAloG ltd. All rights reserved.
+//
+
+@import UIKit;
+
+@protocol UZTScanViewModelProtocol
+
+- (void)didStartInitializingScanView;
+- (void)didInitializeScanView;
+- (void)didReadQRCode:(NSString*)code;
+
+@end
```

Comment:
It looks like overkill here. In this case we will have 3 layers of objects. And win nothing.